$NetBSD: patch-aa,v 1.11 1999/11/22 20:27:23 jdolecek Exp $

The libtool staff was pulled into KDE, as rev 1.331 of
	kde-common/admin/acinclude.m4.in (jdolecek)

--- configure.orig	Wed Apr 28 17:24:12 1999
+++ configure	Thu Jun  3 19:26:48 1999
@@ -507,9 +507,9 @@
 # Prefer explicitly selected file to automatically selected ones.
 if test -z "$CONFIG_SITE"; then
   if test "x$prefix" != xNONE; then
-    CONFIG_SITE="$prefix/share/config.site $prefix/etc/config.site"
+    CONFIG_SITE="$prefix/share/kde/config.site $prefix/etc/config.site"
   else
-    CONFIG_SITE="$ac_default_prefix/share/config.site $ac_default_prefix/etc/config.site"
+    CONFIG_SITE="$ac_default_prefix/share/kde/config.site $ac_default_prefix/etc/config.site"
   fi
 fi
 for ac_site_file in $CONFIG_SITE; do
@@ -2034,6 +2034,8 @@
 kde_save_cflags="$CFLAGS"
 CC=$CXX
 CFLAGS="$CXXFLAGS"
+
+if false; then
 # Always use our own libtool.
 LIBTOOL='$(SHELL) $(top_builddir)/libtool --silent'
 
@@ -2267,6 +2269,7 @@
 ${CONFIG_SHELL-/bin/sh} $ac_aux_dir/ltconfig --no-reexec \
 $libtool_flags --no-verify $ac_aux_dir/ltmain.sh $host \
 || { echo "configure: error: libtool configure failed" 1>&2; exit 1; }
+fi
 
 # Redirect the config.log output again, so that the ltconfig log is not
 # clobbered by the next message.
@@ -5376,37 +5379,37 @@
 if test "default" = "default"; then
 
   if test -z "$kde_htmldir"; then
-    kde_htmldir='\$(prefix)/share/doc/HTML'
+    kde_htmldir='\$(prefix)/share/doc/kde/HTML'
   fi
   if test -z "$kde_appsdir"; then
-    kde_appsdir='\$(prefix)/share/applnk'
+    kde_appsdir='\$(prefix)/share/kde/applnk'
   fi
   if test -z "$kde_icondir"; then
-    kde_icondir='\$(prefix)/share/icons'
+    kde_icondir='\$(prefix)/share/kde/icons'
   fi
   if test -z "$kde_sounddir"; then
-    kde_sounddir='\$(prefix)/share/sounds'
+    kde_sounddir='\$(prefix)/share/kde/sounds'
   fi
   if test -z "$kde_datadir"; then
-    kde_datadir='\$(prefix)/share/apps'
+    kde_datadir='\$(prefix)/share/kde/apps'
   fi
   if test -z "$kde_locale"; then
-    kde_locale='\$(prefix)/share/locale'
+    kde_locale='\$(prefix)/share/kde/locale'
   fi
   if test -z "$kde_cgidir"; then
-    kde_cgidir='\$(exec_prefix)/cgi-bin'
+    kde_cgidir='\$(exec_prefix)/libexec/kde-cgi-bin'
   fi
   if test -z "$kde_confdir"; then
-    kde_confdir='\$(prefix)/share/config'
+    kde_confdir='\$(prefix)/share/kde/config'
   fi
   if test -z "$kde_mimedir"; then
-    kde_mimedir='\$(prefix)/share/mimelnk'
+    kde_mimedir='\$(prefix)/share/kde/mimelnk'
   fi
   if test -z "$kde_toolbardir"; then
-    kde_toolbardir='\$(prefix)/share/toolbar'
+    kde_toolbardir='\$(prefix)/share/kde/toolbar'
   fi
   if test -z "$kde_wallpaperdir"; then
-    kde_wallpaperdir='\$(prefix)/share/wallpapers'
+    kde_wallpaperdir='\$(prefix)/share/kde/wallpapers'
   fi
   if test -z "$kde_bindir"; then
     kde_bindir='\$(exec_prefix)/bin'
